app/src/main/java/org/schabi/newpipe/database/playlist/model/PlaylistRemoteEntity.java

```diff
@@ -132,6 +132,10 @@ public String getUploader() {
         return uploader;
     }
 
+    public String getShortedUploaderName() {
+        return uploader.substring(0, Math.min(uploader.length(), 35)) + "...";
```

Comment:
If I'm not mistaken, this will always show the ellipsis, which is definitely not something that we want.